.gitignore

```diff
@@ -4,8 +4,5 @@
 # Ignore Gradle build output directory
 build
 
-# Ignore wrapper jar
-gradle-wrapper.jar
```

Comment:
I don't think we should have the gradle wrapper at all. Because we only expect a casual user to run the consumer through Docker, no one that would not be able to generate the gradle wrapper themselves will ever need it.

Reply:
The Gradle Wrapper has been removed!